packages/core/src/config/config.ts

```diff
@@ -243,6 +243,8 @@ export const createConfig = <
     } else {
       Object.entries(contract.network).forEach(
         ([networkName, contractOverride]) => {
+          if (!contractOverride) return;
```

Comment:
If you're doing Object.entries, in what scenario would contractOverride be falsy?

Reply:
```ts
network: {
  mainnet: undefined
}
```

Is technically allowed because of the Partial<> type in TypeScript. Not sure how to get around this.